### PR TITLE
Add Beta deploy target

### PR DESCRIPTION
**Pull request by Julien-Molina**

### CODE REVIEW

File: checkTag.sh [hunk 1]
```diff
@@ -2,7 +2,11 @@
 version=`git tag -l --points-at HEAD`
 
 if [ ! -z "$version" ]; then
-	echo "OK"
+	if [[ $string == *"-BETA" ]]; then
+		echo "BETA"
+	else
+		echo "OK"
```
Comment (maurelio1234) [inline checkTag.sh]: OK?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "reactivexcomponent.js",
-  "version": "3.0.1",
+  "version": "3.0.0",
```
Comment (maurelio1234) [inline package.json]: why?

Comment (Julien-Molina) [reply package.json]: 3.0.0 doesn't exist

File: checkTag.sh (hunk 1 above)
Comment (Julien-Molina) [reply checkTag.sh]: You prefer RELEASE ?

File: package.json (hunk 2 above)
Comment (maurelio1234) [reply package.json]: ok 👍

File: checkTag.sh (hunk 1 above)
Comment (maurelio1234) [reply checkTag.sh]: At least it is explicit

**Pull request merged**